services/api/src/resources/project/resolvers.js

```diff
@@ -193,6 +193,68 @@ const getProjectByName = async (
   }
 };
 
+
+const getProjectsByMetadata = async (
+  root,
+  args,
+  {
+    sqlClient,
+    hasPermission,
+  },
+) => {
+
+  let where;
+  try {
+    await hasPermission('project', 'viewAll');
+
+    where = whereAnd([
+      args.createdAfter ? 'created >= :created_after' : '',
+      args.gitUrl ? 'git_url = :git_url' : '',
+    ]);
```

Comment:
you don't accept `createdAfter` and `gitUrl` as parameters for the `projectsByMetadata` Query in GraphQL, so we don't need to check for them here.